pages/events/_id/dashboard.vue

```diff
@@ -5,7 +5,7 @@
     v-else-if="!can('edit', 'posts', item)"
     class="mt-4 mx-auto max-w-md p-4 text-sm text-center"
   >
-    Only event owner can access this area.
+    {{$t("EventDashboard.main")}}
```

Comment:
Do you have Prettier extension installed? There are spaces missing. Prettier will format it for you. Just install it.

```suggestion
    {{ $t("EventDashboard.main") }}
```

Reply:
> Do you have Prettier extension installed? There are spaces missing. Prettier will format it for you. Just install it.

Yes it is installed, now I should format document with it now onwards